image/templates/helm/stackrox-secured-cluster/values-scanner-v4-indexer.yaml.example

```diff
@@ -0,0 +1,106 @@
+# # NOTE:
+# # The Scanner v4 Indexer is not available in the secured clusters installed via roxctl.
+# # The Scanner v4 Matcher is never part of secured clusters, it is only installed as part
+# # of central-services.
+#
+# # Public configuration options for the StackRox Scanner v4 Indexer:
+# # To run Scanner v4 in the secured cluster, you must connect the Scanner v4 to Sensor.
+#
+# # WARNING:
+# #
+# # Installing in the same namespace as Central is unsupported. In that environment,
+# # Sensor will use the existing Scanner v4 Indexer deployed via CentralServices to scan cluster-local and delegated images.
+#
+# Image configuration for scanner v4:
+# # For a complete example, see the `values-public.yaml.example` file.
+# image:
+#  # Configuration of the `scannerV4` image that is used by Scanner v4 Indexer.
+#  scannerV4:
+#    registry: null
+#    name: null
+#    tag: null
+#    fullRef: null
+#  # Configuration of the `scannerV4DB` image that is used by Scanner v4 DB.
+#  scannerV4DB:
```

Comment:
Does this exist?

Reply:
Yes, we have tests for this: https://github.com/stackrox/stackrox/blob/397e82f44585ecd1693b1ac8fa422add8a9f92ee/pkg/helm/charts/tests/securedclusterservices/testdata/helmtest/scanner-v4.test.yaml#L43
I can imagine that the confusion here arises due to the secured-cluster chart using a different image configuration structure (for historical reasons) than the central-services chart.
We are making the connection here: https://github.com/stackrox/stackrox/blob/397e82f44585ecd1693b1ac8fa422add8a9f92ee/image/templates/helm/stackrox-secured-cluster/templates/_init.tpl.htpl#L298
(see the comment above it).